DBM-Naxx/PlagueQuarter/Loatheb.lua

```diff
@@ -57,9 +72,13 @@ function mod:SPELL_CAST_SUCCESS(args)
 		warnDoomNow:Show(doomCounter)
 		timerDoom:Start(timer, doomCounter + 1)
 	elseif args:IsSpellID(55593) then
-		timerAura:Start()
+		if mod:IsDifficulty("normal10") then
 		warnHealSoon:Schedule(14)
 		warnHealNow:Schedule(17)
+		else
+		
+		end
```

Comment:
Indent plz if not revisited later

Reply:
I do be oofing around and forgetting to delete some of my play parts, should be cleaner now